product_docs/docs/pgd/5/routing/proxy.mdx

```diff
@@ -54,6 +54,23 @@ PGD CLI provides a few commands to manage proxies in a PGD cluster, such as `cre
 
 See [Connection management](../routing) for more information on the PGD side of configuration and management of PGD Proxy.
 
+### Proxy health check
+
+PGD Proxy provides the following HTTP(s) health check API endpoints. These APIs are `GET` requests and do not require a request body. See [Configurations](installing_proxy#configuring-health-check) for more details about enabling and configuring this.
+
+```
+GET /health/is-ready
+GET /health/is-live
+```
+
+#### Readiness
+
+On receiving valid request, proxy checks if it can successfully route connections to the current write leader. If the check returns successfully then API returns `true` with HTTP status code `200 (OK)` else `false` with HTTP status code `500 (Internal Server Error)`.
```

Comment:
What makes a invalid request? "On receiving a request, the proxy..."

And is that 'true' because 200 was returned or is there a body containing 'true' returned? It reads like both things happen.

Reply:
```suggestion
On receiving valid 'GET' request, the proxy checks if it can successfully route connections to the current write leader. If the check returns successfully then API gives a response with a body containing `true` and a HTTP status code `200 (OK)`. Otherwise, it returns a body containing `false` with the HTTP status code `500 (Internal Server Error)`.
```

Reply:
Sounds good. Changing it.